target/linux/realtek/files-6.12/drivers/net/pcs/pcs-rtl-otto.c

```diff
@@ -2254,7 +2258,7 @@ static int rtpcs_931x_sds_set_polarity(struct rtpcs_ctrl *ctrl, u32 sds,
 	/* 10GR */
 	val = (tx_val << 1) | rx_val;
 	ret = rtpcs_sds_write_bits(ctrl, sds, 0x6, 0x2, 14, 13, val);
-        if (ret)
+	if (ret)
```

Comment:
might as well fix whitespace in the first commit, since the first commit introduces this function.